Aptoide/aptoide-client-v8
# [ASV-2060| ASV-2083] Downloads and apps improvements

**What does this PR do?**

   This PR contains all downloads improvements. Apps was re-done using epoxy and some fixes were applied to InstallManager, to try and optimize it in order to fix the problem related with the installation being stuck at 100% and performance issues.

**Database changed?**

 No

**Where should the reviewer start?**


**How should this be manually tested?**

  Open the app and perform downloads. Check the interactions with the notification, installations, etc. Also check Apps BN section and see if everything works as expected.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2060](https://aptoide.atlassian.net/browse/ASV-2060)
[ASV-2083](https://aptoide.atlassian.net/browse/ASV-2083)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
